src/mixer/playermanager.cpp

```diff
@@ -580,8 +580,17 @@ void PlayerManager::slotLoadTrackToPlayer(TrackPointer pTrack, QString group, bo
     }
 
     mixxx::Duration elapsed = m_cloneTimer.restart();
-    if (m_lastLoadedPlayer == group && elapsed < mixxx::Duration::fromSeconds(0.5)) {
-        // load pressed twice quickly, clone instead of loading
+    // If not present in the config, use & set the default value
+    if (!m_pConfig->exists(ConfigKey("[Controls]","CloneDeckOnLoadDoubleTap"))) {
+        bool cloneOnDoubleTap = true;
+        m_pConfig->setValue(ConfigKey("[Controls]", "CloneDeckOnLoadDoubleTap"), true);
+    } else {
+        bool cloneOnDoubleTap = m_pConfig->getValue<bool>(
+                    ConfigKey("[Controls]", "CloneDeckOnLoadDoubleTap"));
+    };
```

Comment:
This whole if else block can be simlified by 
```
    bool cloneOnDoubleTap = m_pConfig->getValue(
            ConfigKey("[Controls]", "CloneDeckOnLoadDoubleTap"), true);
```

Reply:
done.